lib/api/api.js

```diff
@@ -50,11 +50,18 @@ const writeContinue = require('../utilities/writeContinue');
 const validateQueryAndHeaders = require('../utilities/validateQueryAndHeaders');
 const parseCopySource = require('./apiUtils/object/parseCopySource');
 
+const actionMap = policies.actionMaps.allActionMap;
+
 auth.setHandler(vault);
 
 /* eslint-disable no-param-reassign */
 const api = {
     callApiMethod(apiMethod, request, response, log, callback) {
+        log.addDefaultFields({
+            service: 's3',
+            action: actionMap[apiMethod],
```

Comment:
What happens if the apiMethod is not in the actionMap keys?

Reply:
@scality-gdoumergue It defaults to the most generic api name for the http method. For example, a 'DeleteBucketEncryption' call gets displayed as 's3:DeleteBucket' or a 'GetObjectTorrent' call is displayed as 's3:GetObject'. Is that acceptable behavior, or would it be better to leave out the action if it's not supported?